src/selectors/entities/posts.js

```diff
@@ -263,3 +263,17 @@ export const getSearchResults = createSelector(
         return postIds.map((id) => posts[id]);
     }
 );
+
+export function makeGetMessageInHistoryItem(index) {
+    return createSelector(
+      (state) => state.entities.posts.messagesHistory,
+      (messagesHistory) => {
```

Comment:
Can you give me a bit of context on how this is to be used? Is it common that a selector will be created for a specific index and used over and over? I don't think a selector makes much sense when the index being asked for is going to change a lot

Reply:
Sorry, probably i have to change the parameter name, is not really the index, is the type.

The idea of usage is create a selector for a specific type ("post" or "comment") and always select the current selectect history entry.

Reply:
Ahh ok, that makes more sense